admin/views/tabs/dashboard/integrations.php

```diff
@@ -0,0 +1,67 @@
+<?php
+/**
+ * WPSEO plugin file.
+ *
+ * @package WPSEO\Admin\Views
+ *
+ * @uses Yoast_Form $yform Form object.
+ */
+
+if ( ! defined( 'WPSEO_VERSION' ) ) {
+	header( 'Status: 403 Forbidden' );
+	header( 'HTTP/1.1 403 Forbidden' );
+	exit();
+}
+
+$integration_toggles = Yoast_Integration_Toggles::instance()->get_all();
+
+?>
+	<h2><?php esc_html_e( 'Integrations', 'wordpress-seo' ); ?></h2>
+	<div class="yoast-measure">
+		<?php
+		echo sprintf(
+		/* translators: %1$s expands to Yoast SEO */
+			esc_html__( '%1$s integrates with a couple of third parties. You can enable or disable these integrations below.', 'wordpress-seo' ),
+			'Yoast SEO'
+		);
+
+		foreach ( $integration_toggles as $integration ) {
+			$help_text = esc_html( $integration->label );
+			if ( ! empty( $integration->extra ) ) {
+				$help_text .= ' ' . $integration->extra;
+			}
+			if ( ! empty( $integration->read_more_label ) ) {
```

Comment:
I'd suggest adding a line break above both `if` statements to improve readability.

Reply:
Makes sense!